Gemfile.lock

```diff
@@ -38,14 +38,14 @@ GEM
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
-    addressable (2.6.0)
-      public_suffix (>= 2.0.2, < 4.0)
+    addressable (2.7.0)
+      public_suffix (>= 2.0.2, < 5.0)
```

Comment:
Question: Are all these Gemfile.lock changes needed?

Reply:
Those just happened through running bundle update. I didn't exactly have control over what was updated

Reply:
Ah ok. If all the tests pass it's fine.